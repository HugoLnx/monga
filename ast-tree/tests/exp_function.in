void main() {
	x = func(10, 20);
}
