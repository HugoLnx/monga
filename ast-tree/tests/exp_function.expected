declarations:
  declaration:
    function:main,262,0
      block:
        statements:
          statement:
            attribution:
              var:x
              exp:10
                function call:func
									exp list:
										exp:0,10
										exp:0,20
